### DISCUSSION TITLE
Failure to get the Keycloak Cluster Up with updated Cluster definition

### DISCUSSION THREAD

**Discussion by gjain-sag** [keycloak/keycloak#12026] [Archive]
Hi Team,

Added below spec to Keycloak cluster definition, but the cluster fails to come up. Any pointers what could be wrong here.

`  unsupported:
    podTemplate:
      spec:
        containers:
          - livenessProbe:
              exec:
                command:
                  - curl
                  - silent
                  - http://127.0.0.1:8080/health/live
              httpGet:
                path: /auth/health/live
                scheme: http
                host:  127.0.0.1
                port: 8080  `

**Comment by andreaTP** [comment]
Hi @gjain-sag , are you missing to include the "below spec"?

Could you elaborate and share the entire Keycloak CR, what you are observing happening in the cluster, and any relevant log?
